examples/amp-story/consent.html

```diff
@@ -68,7 +68,9 @@
           <script type="application/json">{
             "title": "Headline.",
             "message": "This is some more information about this choice. Here's a list of items related to this choice.",
-            "vendors": ["Item 1", "Item 2", "Item 3", "Item 4", "Item 5", "Item 6", "Item 7", "Item 8", "Item 9", "Item 10"]
+            "vendors": ["Item 1", "Item 2", "Item 3", "Item 4", "Item 5", "Item 6", "Item 7", "Item 8", "Item 9", "Item 10"],
+            "externalLinkTitle": "Privacy settings",
+            "externalLink": "https://example.com"
```

Comment:
nit: maybe include `href` in the name?

Reply:
Or `url`.  Is there precedent in `amp-consent` around the naming?  If not, maybe it makes sense to align to the bookend json.

Reply:
`amp-consent` uses `checkConsentHref`, so it may make more sense for a user writing the consent configuration. Plus, it's really a `href`.
I don't feel super strongly about this, let me know if you'd like me to change `href` for `url`.

New config:

````
"externalLink": {
  "href": "https://example.com",
  "title": "Click me"
}
````

PTAL